### ISSUE TITLE
Unable to start rancher desktop

### ISSUE THREAD

**Issue by MastaG** [rancher-sandbox/rancher-desktop#1856]
### Actual Behavior

```Error: /opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255```

Followed by:
```
2022-03-21T14:25:12.516Z: Running command limactl shell --workdir=. 0 mkdir -p bin...
2022-03-21T14:25:12.544Z: Running command limactl copy /tmp/rd-k3s-install-cO4rBc/install-k3s 0:bin/install-k3s...
/usr/bin/scp: Connection closed
2022-03-21T14:25:12.577Z: + limactl copy /tmp/rd-k3s-install-cO4rBc/install-k3s 0:bin/install-k3s
2022-03-21T14:25:12.577Z: Error: /opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
2022-03-21T14:25:12.578Z: Error starting lima: Error: /opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
    at ChildProcess.<anonymous> (/opt/rancher-desktop/resources/app.asar/dist/app/background.js:1:8692)
    at ChildProcess.emit (node:events:394:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```

### Steps to Reproduce

Just start it for the first time.

### Result

The above error message

### Expected Behavior

No errors

### Additional Information

Tried both the RPM and AppImage.
Also tried both containerd and moby.

### Rancher Desktop Version

1.1.1

### Rancher Desktop K8s Version

none

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Other Linux

### Operating System / Build Version

Fedora 36 x86_64

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

rpm

**Comment by joemccall86** [comment]
I will add as a second data point that I'm seeing this exact issue, but only after upgrading from Fedora 35 to Fedora 36. I'm running 1.2.1.

**Comment by adamkpickering** [comment]
Note the following from the [docs](https://docs.rancherdesktop.io/getting-started/installation#installing-via-rpm-package):

> Note: RHEL, Fedora, and related distributions package QEMU, which Rancher Desktop uses on Linux, differently than other distributions. To use Rancher Desktop on these distributions, please use the AppImage.

From the paths in the logs you posted, it would seem that you are running RD from an rpm. This isn't a supported way of installing RD on Fedora. Is the error the same when running from an AppImage?

I tried to reproduce this in a Fedora 36 (beta) VM, and I get one of two errors, the first being more common:
```
 * /var/log/containerd.log: creating file
 * Starting Container Daemon ... [ ok ]
/usr/bin/scp: Connection closed
2022-04-01T17:32:00.345Z: + limactl copy /tmp/rd-k3s-install-Uh5mvB/install-k3s 0:bin/install-k3s
2022-04-01T17:32:00.345Z: Error: /tmp/.mount_ranchesihcqW/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
2022-04-01T17:32:00.351Z: Error starting lima: Error: /tmp/.mount_ranchesihcqW/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
    at ChildProcess.<anonymous> (/tmp/.mount_ranchesihcqW/opt/rancher-desktop/resources/app.asar/dist/app/background.js:1:8836)
    at ChildProcess.emit (node:events:394:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```
```
* rc-service: service `k3s' does not exist
2022-04-01T17:34:29.551Z: + limactl shell --workdir=. 0 sudo /sbin/rc-service --ifstarted k3s stop
2022-04-01T17:34:29.551Z: Error: /tmp/.mount_ranche73IGxp/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 1
2022-04-01T17:34:29.552Z: Error starting lima: Error: /tmp/.mount_ranche73IGxp/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/tmp/.mount_ranche73IGxp/opt/rancher-desktop/resources/app.asar/dist/app/background.js:1:8836)
    at ChildProcess.emit (node:events:394:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```

**Comment by timlinux** [comment]
I'm getting the same issue here on F36 beta here. I am using the appimage as recommended and have no RPM-based installation currently or in the past. I can replicate the issue on two hosts. I include the logs below as they are slightly different from the post above by the looks of things.

Error starting kubernetes:

```bash
rror: /tmp/.mount_ranchehSrZKO/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
```


Last command run:

```bash
/tmp/.mount_ranchehSrZKO/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl copy /tmp/rd-k3s-install-p1YbMJ/install-k3s 0:bin/install-k3s
```



```bash

 * /var/log/containerd.log: creating file
 * Starting Container Daemon ... [ ok ]
/usr/bin/scp: Connection closed
2022-05-02T14:41:39.501Z: + limactl copy /tmp/rd-k3s-install-p1YbMJ/install-k3s 0:bin/install-k3s
2022-05-02T14:41:39.501Z: Error: /tmp/.mount_ranchehSrZKO/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
2022-05-02T14:41:39.502Z: Error starting lima: Error: /tmp/.mount_ranchehSrZKO/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 255
    at ChildProcess.<anonymous> (/tmp/.mount_ranchehSrZKO/opt/rancher-desktop/resources/app.asar/dist/app/background.js:17:141690)
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```

Additionally I checked the four lima related log files under 'troubleshooting'. 

![image](https://user-images.githubusercontent.com/178003/166258770-7e0f509e-253f-4f7e-9802-6cfc4f21b680.png)

lima.ha.sterr.log is attached here in the zip

limage.ha.stdout.log shows:

```bash
{"time":"2022-05-02T15:41:18.949851953+01:00","status":{"sshLocalPort":46035}}
{"time":"2022-05-02T15:41:39.282149568+01:00","status":{"running":true,"sshLocalPort":46035}}
```


The first log listed is empty. The second is attached in the zip for your reference.
[rancher-lima-logs.zip](https://github.com/rancher-sandbox/rancher-desktop/files/8603925/rancher-lima-logs.zip)

**Comment by wilf1rst** [comment]
I will add myself to the list of broken Rancher Desktop on Fedora. I was using Rancher Desktop since it was announced and it's unusable since I upgraded to Fedora 36. I previously used the RPM based solution, which I can't upgrade to RD 1.3 on fedora 36 because of dependancies issues so I uninstall it and switch to AppImage, then reset to factory but with the same results as others mentioned.

Main error:
```
Error: /tmp/.mount_ranchefKcmDh/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 1
```

Logs:
```
2022-05-11T13:13:55.459Z: + limactl --debug shell --workdir=. 0 sudo /sbin/rc-service --ifstarted k3s stop
2022-05-11T13:13:55.460Z: Error: /tmp/.mount_ranchefKcmDh/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 1
time="2022-05-11T15:14:16+02:00" level=debug msg="changeDirCmd=\"cd . || exit 1\""
time="2022-05-11T15:14:16+02:00" level=debug msg="OpenSSH version 8.8.1 detected"
time="2022-05-11T15:14:16+02:00" level=debug msg="AES accelerator seems available, prioritizing aes128-gcm@openssh.com and aes256-gcm@openssh.com"
time="2022-05-11T15:14:16+02:00" level=debug msg="executing ssh (may take a long)): [/usr/bin/ssh -F /dev/null -o IdentityFile=\"/home/wilfried/.local/share/rancher-desktop/lima/_config/user\" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o NoHostAuthenticationForLocalhost=yes -o GSSAPIAuthentication=no -o PreferredAuthentications=publickey -o Compression=no -o BatchMode=yes -o IdentitiesOnly=yes -o Ciphers=\"^aes128-gcm@openssh.com,aes256-gcm@openssh.com\" -o User=wilfried -o ControlMaster=auto -o ControlPath=\"/home/wilfried/.local/share/rancher-desktop/lima/0/ssh.sock\" -o ControlPersist=5m -q -p 42503 127.0.0.1 -- cd . || exit 1 ; exec \"$SHELL\" --login -c 'sudo /sbin/rc-service --ifstarted k3s stop']"
 * rc-service: service `k3s' does not exist
2022-05-11T13:14:16.771Z: + limactl --debug shell --workdir=. 0 sudo /sbin/rc-service --ifstarted k3s stop
2022-05-11T13:14:16.771Z: Error: /tmp/.mount_ranchefKcmDh/opt/rancher-desktop/resources/resources/linux/lima/bin/limactl exited with code 1
```

**Comment by adamkpickering** [comment]
Thanks everyone for your patience. We figured out that the problem was caused by openssh version 8.8. There are two ways that `scp` can work: calling the `scp` command on the remote host via the SSH protocol, or via SFTP. The openssh project has deprecated the former. While it doesn't say so in the release notes for openssh 8.8, it appears that `scp` now uses SFTP by default.

This means that the remote host (the RD VM in our case) must have an SFTP server installed. This wasn't the case, so we weren't able to copy certain files into the VM that needed to be there. This resulted in an inability to start the VM. We've now added an SFTP server to the VM, so things should work smoothly.

This fix will be in RD 1.4.0, which we expect to release within the next week or so. In the meantime, there is a workaround in #2139. Closing this issue for now - please comment if you continue to experience issues.
